README.md

```diff
@@ -1,6 +1,56 @@
-# [ng-suggestion](http://ianwalter.github.io/ng-suggestion/)
+# [ng-suggestion](https://github.com/flashnotes/ng-suggestion/)
 *Flexible AngularJS typeahead / autocomplete / suggestion / predictive search directive*
 
-I don't have time for docs right now just read the source.
+#### Dependencies
+* [ng-dropdown](https://github.com/flashnotes/ng-dropdown/)
 
-&mdash;&nbsp; [Ian Kennington Walter](http://iankwalter.com)
+#### Installing ng-suggestion
+
+Install using Bower:  
+```
+bower install ng-suggestion --save
+```
+
+Include js/ng-suggestion.min.js in your app. You may also want to include
+css/ng-suggestion.min.css, but it's not necessary, only recommended.
```

Comment:
Let's put this information in the "Setup" or "Using" sections. How about showing sample markup to include those files?

``` html
<link rel="stylesheet" media="screen" href="/bower_components/ng-suggestion/dist/css/ng-suggestion.min.css" />
```

``` html
<script src="/bower_components/ng-suggestion/dist/js/ng-suggestion.min.js"></script>
```